coverage_config_x86_64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 91.7,
+  "coverage_score": 91.8,
```

Comment:
The coverage number probably needs to be updated since #11 was merged.

Reply:
Hi, Andreea is on vacation. I just pushed the required coverage update and added one commit with a few comments regarding the safety of some operations.
I'm still not sure if [this](https://github.com/rust-vmm/vm-fdt/blob/master/src/writer.rs#L241) can never panic, I think I would still use `checked_add` here :-?. Thoughts?